source/fixed_array.rb

```diff
@@ -1,3 +1,25 @@
 class FixedArray
+  attr_accessor :array
```

Comment:
Make it private! If you think you need it you need to re-think how you're using FixedArray